clash-prelude/src/Clash/Explicit/Prelude.hs

```diff
@@ -178,6 +178,19 @@ import Clash.XException
 >>> let windowD3 = windowD @2
 -}
 
+{- $hiding
+"Clash.Explicit.Prelude" re-exports most of the Haskell "Prelude" with the
+exception of the following: @(++)@, @(!!)@, @concat@, @drop@, @even@, @foldl@,
+@foldl1@, @foldr@, @foldr1@, @head@, @init@, @iterate@, @last@, @length@,
+@map@, @odd@, @repeat@, @replicate@, @reverse@, @scanl@, @scanr@, @splitAt@,
+@tail@, @take@, @unzip@, @unzip3@, @zip@, @zip3@, @zipWith@, @zipWith3@.
+
+It instead exports the identically named functions defined in terms of 'Vec' at
+"Clash.Sized.Vector". For the 'Clash.Class.Parity.odd' and
+'Clash.Class.Parity.even' function a type class called
+'Clash.Class.Parity.Parity' is available at "Clash.Class.Parity".
+-}
```

Comment:
Should this (and the others) also document the changed `&&` and `||` from https://github.com/clash-lang/clash-compiler/pull/1900 ?

It's the same function, so maybe not?

Reply:
It's the same function, so in that sense I don't think it needs doc. However, if you look at [HaskellPrelude](https://hackage.haskell.org/package/clash-prelude-1.5.0/candidate/docs/Clash-HaskellPrelude.html#v:-38--38-), you'll see that it documents everything it re-exports, but omits the docs for the redefinitions `&&`, `||` and `not`. That's not really very nice, but seems pretty low priority.